lib/Thingiverse.pm

```diff
@@ -60,8 +52,19 @@ our $client_id          = 'c587f0f2ee04adbe719b';
 our $access_token       = 'b053a0798c50a84fbb80e66e51bba9c4';
```

Comment:
@perlygatekeeper I want to turn these 'our' objects into required attributes in the class and get rid of these hard-coded access tokens.  If you want to test with real access tokens locally, I would recommend that you add a gpg encrypted file locally or to some test are and use it to load up stuff.  Then, around your tests you would skip if the person couldn't decrypt the file.